src/components/Table/types.ts

```diff
@@ -37,12 +37,28 @@ export type columnTypes = Columns;
 /** The data for a single row. */
 export type RowEntry = Record<string, any>;
 
+export interface SortState {
+  /** The key of the header column that is currently being sorted by. */
+  sortedColumn?: string;
```

Comment:
if it's not too late, we could change this to `sortedColumnKey` and get rid of the comment